### PR TITLE
Remove old diags folder if a run is not restarted from a checkpoint

### PR DESCRIPTION
**Pull request by roelof-groenewald**

### CODE REVIEW

File: mewarpx/mewarpx/runinfo.py [hunk 1]
```diff
@@ -30,14 +30,14 @@ class SimInfo(object):
     This must contain:
         SimInfo.nxyz (int nx, int ny, int nz)
         SimInfo.pos_lims (floats xmin, xmax, ymin, ymax, zmin, zmax)
-        SimInfo.geom (str 'XZ', 'RZ' or 'XYZ')
+        SimInfo.geom (str 'X', 'XZ', 'RZ' or 'XYZ')
```
Comment (KZhu-ME) [inline mewarpx/mewarpx/runinfo.py]: Is the 1D geom string "X"? If so, should we make it consistent with warpx since, warpx uses Z for 1D?

Comment (roelof-groenewald) [reply mewarpx/mewarpx/runinfo.py]: Haha no it is `Z`. I have no idea why I put `X` there.

**Review by peterscherpelz — APPROVED** [review]
Looks good, thanks!

File: mewarpx/mewarpx/mwxrun.py [hunk 2]
```diff
@@ -168,24 +170,14 @@ def init_run(self, restart=None, checkpoint_dir="diags",
                     )
                 )
 
+            # at this point we are committed to either restarting or starting
```
Comment (peterscherpelz) [inline mewarpx/mewarpx/mwxrun.py:174]: Make sure this executes only on new runs.

File: mewarpx/mewarpx/mwxrun.py [hunk 3]
```diff
@@ -168,24 +170,14 @@ def init_run(self, restart=None, checkpoint_dir="diags",
...
+            # fresh; if this is a fresh start we can delete diags if present
+            if os.path.exists("diags"):
```
Comment (roelof-groenewald) [inline mewarpx/mewarpx/mwxrun.py]:
```suggestion
            if not self.restart and os.path.exists("diags"):
```

**Pull request merged**